parsers/FR_O.py

```diff
@@ -1,351 +1,188 @@
-        },
-        {
-            "name": "bagasse_charbon",
-            "monthly_variation": True,
-            "type": MAP_GENERATION_BAGASSE_COAL_REUNION,
-        },
-        {
-            "name": "thermique_mwh",
-            "monthly_variation": False,
-            "type": MAP_GENERATION_DIESEL_GAS_REUNION,
-        },
-        {
-            "name": "thermique",
-            "monthly_variation": False,
-            "type": MAP_GENERATION_DIESEL_GAS_REUNION,
-        },
-        # (RE) API update
-        {
-            "name": "turbine_a_combustion",
-            "monthly_variation": False,
-            "type": MAP_GENERATION_REUNION_OIL,
-        },
-        {
-            "name": "diesel",
-            "monthly_variation": False,
-            "type": MAP_GENERATION_REUNION_OIL,
-        },
-        {
-            "name": "charbon",
-            "monthly_variation": False,
-            "type": MAP_GENERATION_REUNION_COAL,
-        },
-    ],
+LIVE_DATASETS = {
+    "FR-COR": "production-delectricite-par-filiere-en-temps-reel",
+    "GP": "mix-temps-reel-guadeloupe",
 }
 
-# Non-thermal sources names in api
-sources_mapping = {
-    "biomass": ["bioenergies_mwh", "bioenergies", "biogaz"],
-    "hydro": ["hydraulique_mwh", "hydraulique", "micro_hydro"],
-    "solar": [
-        "photovoltaique_mwh",
-        "photovoltaique",
-        "photovoltaique_avec_stockage",
-        "photovoltaique0",
-    ],
-    "wind": ["eolien_mwh", "eolien"],
-    "geothermal": ["geothermie_mwh"],
-    "oil": ["moteur_diesel"],
+HISTORICAL_DATASETS = {
+    "FR-COR": "production-delectricite-par-filiere",
+    "RE": "courbe-de-charge-de-la-production-delectricite-par-filiere",
+    "GF": "courbe-de-charge-de-la-production-delectricite-par-filiere",
+    "MQ": "courbe-de-charge-de-la-production-delectricite-par-filiere",
+    "GP": "courbe-de-charge-de-la-production-delectricite-par-filiere",
 }
 
+API_PARAMETER_GROUPS = {
+    "production": {
+        "biomass": [
+            "biomasse",
+            "biomasse_mw",
+            "biomasse_mwh",
+            "bioenergies",
+            "bioenergies_mw",
+            "bioenergies_mwh",
+        ],
+        "coal": [
+            "charbon",
+        ],
+        "geothermal": [
+            "geothermie",
+            "geothermie_mw",
+        ],
+        "hydro": [
+            "hydraulique",
+            "hydraulique_mw",
+            "hydraulique_mwh",
+            "micro_hydro",
+            "micro_hydraulique_mw",
+        ],
+        "oil": ["diesel", "moteur_diesel"],
+        "solar": [
+            "photovoltaique",
+            "photovoltaique0",
+            "photovoltaique_mw",
+            "photovoltaique_mwh",
+            "solaire_mw",
+        ],
+        "wind": [
+            "eolien",
+            "eolien_mw",
+            "eolien_mwh",
+        ],
+        "unknown": [
+            "bagasse_charbon_mwh",
+            "charbon_bagasse_mw",
```

Comment:
charbon is coal, so I think we can categorize those as coal for now and adjust with regional emission factors later when it's released

Reply:
In the past we have put all mixed modes in unknown and then created a adjusted regional carbon intensity for unknown. I'd like to stick with this method if possible as it makes it clear that all biomass is biomass and all coal is coal etc.

This is how we (I) did it for Sweden for example:
```yaml
    unknown:
      - _comment: Uses the 2015 (2022 edition) eurostat production values for the sources
          in unknown
        _url: https://ec.europa.eu/eurostat/documents/38154/4956218/Energy-Balances-January-2022-edition.zip/2dc71f1b-806a-b9ed-e77d-67b7c705197b?t=1643660477829
        datetime: '2015-01-01'
        source: assumes 85.46% biomass, 7.98% gas, 2.77% coal, 1.8% oil, 1.3% peat,
          0.69% solar
        value: 67.7
```